addon-test-support/key-event.js

```diff
@@ -1,11 +1,16 @@
 import { triggerEvent } from './trigger-event';
+import Ember from 'ember';
 
-/*
-  @method keyEvent
-  @param {String|HTMLElement} selector
-  @param {String} type
-  @param {Number} keyCode
-*/
-export function keyEvent(selector, type, keyCode) {
-  return triggerEvent(selector, type, { keyCode, which: keyCode });
+const { merge } = Ember;
+
+/**
+ * @public
+ * @param selector
+ * @param type
+ * @param keyCode
+ * @param modifiers
+ * @return {*}
+ */
+export function keyEvent(selector, type, keyCode, modifiers = { ctrlKey: false, altKey: false, shiftKey: false, metaKey: false }) {
```

Comment:
I'm 100% for this as before it wasn't possible to simulate `ctrl + A` and things like that. Can you edit the readme and add docs for this? Perhaps it's easier if you improve `keyEvent` in a separate PR.